test/project.py

```diff
@@ -598,3 +598,20 @@ def add_custom_roles(project, roles):
 
     with open(init_filepath, 'w') as f:
         f.write(new_init)
+
+
+def set_whoami_build_specs(rockspec_path, project_name, version):
+    who_am_i = 'I am %s' % rockspec_path
+
+    with open(rockspec_path, 'w') as f:
+        f.write('''
+                package = '{}'
+                version = '{}'
+                source  = {{ url = '/dev/null' }}
+                dependencies = {{ 'tarantool' }}
+                build = {{ type = 'command',
+                          build_command = 'echo {}'}}
+            '''.format(project_name, version, who_am_i))
+        f.close()
+
+    return who_am_i
```

Comment:
Logic of this function is not clear. It does two things at once:
* Creating rockspec file
* Returning whoami string

From the name of the function, it is not obvious that the function returns whoami string.

Reply:
maybe move getting who_am_i to another function, like ``get_rockspec_path`` below?

Reply:
It's very good comment.

Usually such functions are named `doSmthAndReturnSmth`. I don't like moving getting whoami to function, because it's too specific for this test.

What about `set_and_return_whoami_on_build`?